roles/tiflash/templates/run_tiflash_binary.sh.j2

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+ulimit -n 1000000
+
+# WARNING: This file was auto-generated. Do not edit!
+#          All your edit might be overwritten!
+cd "{{ deploy_dir }}" || exit 1
+
+export RUST_BACKTRACE=1
+
+export TZ=${TZ:-/etc/localtime}
+export LD_LIBRARY_PATH={{ deploy_dir }}/bin/tiflash:$LD_LIBRARY_PATH
+
+echo -n 'sync ... '
+stat=$(time sync)
+echo ok
+echo $stat
+
+echo $$ > "status/{{ role_name }}.pid"
```

Comment:
Maybe we should add `mkdir -p status` to ensure dir status exists.

Reply:
Done. `tidb-ansible` can create status dir for all services now.